packages/utils/src/parser/token.ts

```diff
@@ -109,6 +109,7 @@ export interface Token {
   symbol: string;
   name: string;
   decimals: number;
+  logo?: string;
```

Comment:
I have a feeling we're conflating 2 meanings of "token" here.
One is just a unit for an amount, which we use for rendering amounts.
The other is the entire infrastructure required to operate a token, so a ledger, index, etc.
And I think the logo is more part of the second.

Do you think it would make sense to have 2 separate types?
In a sense we already do because in nns-dapp we have [IcrcTokenMetadata](https://github.com/dfinity/nns-dapp/blob/437e919fbbbb22542537a8a1a8f711268460ed62/frontend/src/lib/types/icrc.ts#L9) for this purpose.

Reply:
I actually was thinking on removing that one and using always `Token`.

`name` is also not used here for example.

Another idea would be to remove the `Token` from here altogether and pass directly `symbol` and `decimals` which are the only necessary for rendering the data.

Reply:
`Token` is used in a lot of places. Wouldn't it be a lot of work to replace all of them with 2 separate variables?
It seems easier to make a new type for the "full token configuration" (`TokenConfig`?) and use that in places where we need more than rendering.
Although I'm not very confident and haven't looked through all the places where Token is used.

Anyway, I don't want to block you. But if you want to replace `IcrcTokenMetadata`, you would have to add `fee` here as well. Which is unfortunate because `fee` is not optional in `IcrcTokenMetadata`.

Reply:
Yes, I know. I thought it would be easier and started doing it at some point. Then I found this `fee` field and stopped because it grew in complexity.

But this is also technical debt. I believe we should fix that because it will cause confusion.